pinax/stripe/actions/subscriptions.py

```diff
@@ -48,7 +48,7 @@ def create(customer, plan, quantity=None, trial_days=None, token=None, coupon=No
 
     subscription_params = {}
     if trial_days:
-        subscription_params["trial_end"] = datetime.datetime.utcnow() + datetime.timedelta(days=trial_days)
+        subscription_params["trial_end"] = datetime.datetime.now(tz=timezone.utc) + datetime.timedelta(days=trial_days)
```

Comment:
What about using `django.utils.timezone.now()`?
It seems to be used in other places.

Reply:
Changed another occurrence in https://github.com/pinax/pinax-stripe/pull/577 for consistency.